irohad/model/converters/impl/pb_query_response_factory.cpp

```diff
@@ -235,7 +237,8 @@ namespace iroha {
       PbQueryResponseFactory::deserializeRolePermissionsResponse(
           const protocol::RolePermissionsResponse &response) const {
         model::RolePermissionsResponse res;
-        std::copy(response.permissions().begin(), response.permissions().end(),
+        std::copy(response.permissions().begin(),
+                  response.permissions().end(),
                   res.role_permissions.begin());
```

Comment:
```
std::back_inserter
```
?
as in 
https://github.com/hyperledger/iroha/pull/706/files#diff-54506143c6fcf71e8b5954f1e82bf10fR222